File: 待办清单功能/ruankai2/app/src/main/java/com/byted/camp/todolist/TodoListApplication.java
package com.byted.camp.todolist;

import android.app.Application;
import android.content.Context;

import com.facebook.stetho.Stetho;


public class TodoListApplication extends Application {
    private static Context context;



    @Override
    public void onCreate() {
        super.onCreate();
        Stetho.initializeWithDefaults(this);
        context = getApplicationContext();
    }

    public static Context getContext() {
        return context;
    }
}

File: 12.25整合代码/MyApplication2/app/src/main/java/com/example/myapplication/todolist/TodoListApplication.kt
package com.example.myapplication.todolist

import android.app.Application
import android.content.Context
import com.facebook.stetho.Stetho


class TodoListApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Stetho.initializeWithDefaults(this)
        context = applicationContext
    }

    companion object {
        var context: Context? = null
            private set
    }
}
